android/app-newm/src/main/java/io/newm/accounts/NewmAccountAuthenticator.kt

```diff
@@ -0,0 +1,50 @@
+package io.newm.accounts
+
+import android.accounts.AbstractAccountAuthenticator
+import android.accounts.Account
+import android.accounts.AccountAuthenticatorResponse
+import android.content.Context
+import android.os.Bundle
+
+internal class NewmAccountAuthenticator(context: Context) : AbstractAccountAuthenticator(context) {
```

Comment:
We can go with the basic implementation for now, curious to understand what else can be done here @wlara

Reply:
@cristhianescobar Most of the features are intended for Accounts that go across multiple App (e.g., Google Accounts). For single-app accounts like ours, what we have here is enough.